From 584f0c4f1bb4f615f9f178564323b158c7ce3167 Mon Sep 17 00:00:00 2001
From: Tam Nguyen Duc <TamND8@fsoft.com.vn>
Date: Mon, 1 Oct 2018 16:57:04 +0700
Subject: Reconfig LOOPS_MULTIPLIER to fix issue periodic_cpu_load on iwg1C/E

Description:
	On iwg1C/E boards, because the performance is lower than IWG1H/M/N boards.

	So, this board can not keep the time execute function calc(int loops) on time (default period = 5ms)

	Have to decreasing LOOPS_MULTIPLIER from 4.2 into 4 to keep the time execute function calc(int loops) on time.

Signed-off-by: Tam Nguyen Duc <TamND8@fsoft.com.vn>
---
 testcases/realtime/func/periodic_cpu_load/periodic_cpu_load_single.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/realtime/func/periodic_cpu_load/periodic_cpu_load_single.c b/testcases/realtime/func/periodic_cpu_load/periodic_cpu_load_single.c
index ebbb461..b6a6865 100644
--- a/testcases/realtime/func/periodic_cpu_load/periodic_cpu_load_single.c
+++ b/testcases/realtime/func/periodic_cpu_load/periodic_cpu_load_single.c
@@ -48,7 +48,7 @@
 #define DEFAULT_PERIOD 5
 #define DEFAULT_PRIO   90
 #define DEFAULT_CALC_LOOPS 1000
-#define LOOPS_MULTIPLIER 4.2
+#define LOOPS_MULTIPLIER 4
 #define DEFAULT_FILENAME_PREFIX "pcl"

 static int prio;
--
2.7.4
